test_libs/pyspec/eth2spec/test/helpers/attestations.py

```diff
@@ -0,0 +1,161 @@
+from typing import List
```

Comment:
Direct merges from `helpers.py` generally failed due to splitting across files.
Fairly confident in the few things I had to fix up due to the tests passing :)